eq-author/src/App/publish/PublishPage.js

```diff
@@ -0,0 +1,220 @@
+import React, { useState, useEffect } from "react";
```

Comment:
I had a bit of a look at this, thought it might look a bit nicer using hooks and also get to get to lose the query component, take a look and let me know what you think. The change will break a couple tests though sorry!!

https://gist.github.com/SamGodwin2/fbb7c6b5a0f468e822f42a4be717769d